developers/bindings/semantic-tags.md

```diff
@@ -0,0 +1,73 @@
+
+## General Rules for Semantic Tags
+
+The various equipment, points, properties, and locations are displayed in the openHAB User Interface in a hierarchical tree structure, which is based on a hierarchy of semantic tags defined in openHAB Core. The list of semantic tags defined in openHAB Core is extensible. This document describes the developer rules for applying or extending this list.
+
+Note: the core predefined semantic tags are presented to users as propositions, suggestions, or hints. It is possible for users to override or customise such tags at their own discretion. The remainder of this document does NOT relate to user overrides or customisation. It applies solely to the openHAB Core tags.
+
+### Rules for **APPLYING** Core Predefined Semantic Tags
+
+1. Developers MUST apply core predefined tags in their addons where possible.
+1. Developers MAY hard code the tags in the ‘thing-type.xml’ or apply them dynamically at run time.
+1. Developers SHALL NOT hard code LOCATION tags in the ‘thing-type.xml’.
+1. Developers MAY apply LOCATION tags dynamically at run time (if the addon can determine the location).
+1. Developers SHALL NOT apply overridden or customised tags in their addons.
+1. Developers MAY apply a POINT tag without a PROPERTY tag (in cases where a property makes no sense). However, in such a case the developer MAY submit a new PROPERTY tag that fits.
+1. Developers SHALL NOT apply a PROPERTY tag without a POINT tag.
+1. Semantic EQUIPMENT tags MUST be applied at thing level.
+1. Semantic POINT and PROPERTY tags MUST be applied at channel level.
+1. Developers MAY open a new PR to propose extensions to the core predefined tags as follows.
+
+### Rules for **EXTENDING** Core Predefined Semantic Tags
+
+11. The purpose of the semantic model is to maintain a stable structure, so it is intended that only rarely shall new tags be submitted or adopted.
+1. New tags MUST fit within the hierarchy of semantic tags already defined in openHAB Core.
+1. New tags SHALL NOT be synonyms of existing tags e.g. kitchen/kitchenette, sitting/living room.
+1. New tags MUST prove that PROPERTY, POINT and EQUIPMENT tags could be applied to three or more bindings.
+1. Developers MUST provide supporting evidence in their PR that the rules above and below are met.
+
+## Equipment Tags
+
+An EQUIPMENT tag describes the equipment type of a piece of equipment. Examples are light bulb, thermostat, pump, car, wall switch, push button, alarm system, or alarm detector.
+
+16. New EQUIPMENT tags MUST describe the equipment type of a physical or virtual device.
+1. New EQUIPMENT tags MUST be a noun.
+
+## Point Tags
+
+A POINT is a tag that describes the functional type (or purpose) of a data point within an equipment. Examples are measurement, control, set point, or status.
+
+18. New POINT tags MUST describe the functional type (or purpose) of the data point.
+1. New POINT tags MUST be what you do to, or how you get, the state of the Item.
```

Comment:
We might need to elaborate. I'm not sure this is clear enough. Then again, the examples from above may be sufficient.

Reply:
> We might need to elaborate.

Any proposals?

Reply:
I don't have anything yet that isn't going to be overly long. 

Maybe we just list a couple of examples.

> , e.g. a sensor reports a Measurement, a thermostat's target temperature is a Setpoint, etc.